modules/icon/css/lib/utils.js

```diff
@@ -38,12 +38,11 @@ export function getColor(color) {
   if (!/[0-9]{3}$/.test(color)) {
     findColor = `${color}500`;
   }
-
-  const foundColor = Object.entries(canvasColors).find(([colorName]) => colorName === findColor);
+  const foundColor = Object.keys(canvasColors).find(colorName => colorName === findColor);
 
   if (!foundColor) return null;
 
```

Comment:
Nit: since this is just an equals, can we avoid the extra call or is there an advantage to find?

```suggestion
  const foundColor = canvasColors[findColor]

  return (foundColor ? foundColor : null);
```

Reply:
Hah - yep. I just didn't take a close enough look at what I was refactoring. Thanks 👍